components/signup/signup.tsx

```diff
@@ -486,20 +500,24 @@ const Signup = ({onCustomizeHeader}: SignupProps) => {
                             min: Constants.MIN_USERNAME_LENGTH,
                             max: Constants.MAX_USERNAME_LENGTH,
                         },
-                    )
-                ));
+                    );
+                    sendFirstAdminUserTelemetryEvents('provided_username_format_error');
+                }
+                setNameError(nameError);
                 isValid = false;
             }
         } else {
             setNameError(formatMessage({id: 'signup_user_completed.required', defaultMessage: 'This field is required'}));
+            sendFirstAdminUserTelemetryEvents('not_provided_user_name_when_trying_to_create_account');
             isValid = false;
         }
 
         const providedPassword = passwordInput.current?.value ?? '';
-        const {error} = isValidPassword(providedPassword, getPasswordConfig(config), intl);
+        const {error, errorId} = isValidPassword(providedPassword, getPasswordConfig(config), intl);
 
         if (error) {
             setPasswordError(error as string);
+            sendFirstAdminUserTelemetryEvents(`password_error_${errorId}`);
```

Comment:
this password error id event will contain one of the following variations: 
https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/password_settings.jsx#L29